### PR TITLE
View Job Posting Applications Details #54

### PR DESCRIPTION
**Pull request by OussamaJmili**
* Added a new page for Applications viewing 
* The recruiter can now view all applications in details #54 
* The recruiter can view/download applicant's documents
* Styled the Applications page

Job Applications Page Completed.

### CODE REVIEW

File: linkedin-clone/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
-    "firebase": "^9.17.1",
+    "firebase": "^9.17.2",
```
Comment (TNT05) [inline linkedin-clone/package.json:9]: no regression from upgrading version ?

File: linkedin-clone/src/components/Main.js [hunk 2]
```diff
@@ -30,13 +30,13 @@ const Main = (props) => {
 
   const history = useHistory();
 
-  const handleApply = (jobPostingId) => {
-    history.push(`/job-posting/${jobPostingId}`);
-  };
+  // const handleApply = (jobPostingId) => {
+  //   history.push(`/job-posting/${jobPostingId}`);
```
Comment (TNT05) [inline linkedin-clone/src/components/Main.js:34]: applying still not implemented ?

**Review by TNT05 — APPROVED** [review]

File: linkedin-clone/src/components/Main.js (hunk 2 above)
Comment (OussamaJmili) [reply linkedin-clone/src/components/Main.js:34]: applying is implemented in JobPostingPage.js
https://raw.githubusercontent.com/TNT05/MiniCap/main/linkedin-clone/src/components/JobPostingPage.js
The button only takes to the job posting page.

**Review by MomooDiane — APPROVED** [review]
Looks fine to me

**Pull request merged**